package.json

```diff
@@ -10,19 +10,21 @@
     "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
-    "lodash": "^3.10.1",
+    "babel-cli": "^6.26.0",
```

Comment:
@carlospaelinck why was `babel-cli` added to the project? Assuming there is a reason, is it intended to be a dependency and not a devDependency?

Reply:
I added that as an experiment to determine which package was crashing metro-bundler in 50.3. Let me remove.